index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="#">
+    <title>AVS - Home page</title>
+</head>
+<body>
+    <div id="overlay"></div>
+    <div id="mobile-menu" class="mobile-menu-m">
+        <ul>
+            <li><a href="index.html" class="menu-item">Home</a></li>
+            <li><a href="about.html" class="menu-item">About</a></li>
+            <li><a href="#" class="menu-item">Logout</a></li>
+        </ul>
+    </div>
+    <header class="top-section">
+        <div class="top-bar">
+            <div>
+                <nav class="menu-text">
+                    <ul class="nav-links">
+                        <li>
+                            <a id="nav-top" href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                        </li>
+                        <li>
+                            <a id="nav-top" href="#"><i class="fa-brands fa-twitter"></i></a>
+                        </li>
+                        <li><a id="nav-top" class="text-dec" href="index.html">Home</a></li>
+                        <li><a id="nav-top" class="text-dec" href="about.html">About</a></li>
+                        <li><a id="nav-top" class="text-dec" href="#">Logout</a></li>
+                    </ul>
+                </nav>
```

Comment:
Great job @MoyasiGinko 🤝

- [ ] [Optional] Kindly follow the given design as much as possible so that your project can become its close replica. You may align the navbars properly as they appear in the given design.
- Your design
![your navbar](https://user-images.githubusercontent.com/63917725/235011512-e51ec403-0345-49bc-897b-329687f0c0e0.PNG)
- Given design
![given navbar](https://user-images.githubusercontent.com/63917725/235011554-1f817ee9-bb41-41eb-aa3e-acd0e1298095.PNG)